### ISSUE TITLE
cdk.json

### ISSUE THREAD

**Issue by alimeerutech** [aws/aws-cdk#9346]
support adding multiple app names in cdk.json

{
"apps": [
  { 
    "name": "app1",
    "command": "python dir1/app.py"
  },
  {
    "name": "app2",
    "command": "python dir2/app.py"
  }
]
}

### Use Case

If you have multiple apps sharing the same cdk.json you either have to pass in the app via cdk --app or copy cdk.json around, it would be much nicer if we could support having a single cdk.json with multiple apps 

### Proposed Solution

support adding multiple app names in cdk.json

{
"apps": [
  { 
    "name": "app1",
    "command": "python dir1/app.py"
  },
  {
    "name": "app2",
    "command": "python dir2/app.py"
  }
]
}

### Other

* [ ] :wave: I may be able to implement this feature request
* [X] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

**Comment by shivlaks** [comment]
@alimeerutech have you tried putting shared configuration in `~/.cdk.json` as a user setting, or is the use case that you have a group of apps that require specific shared settings and you'd like to be able to specify it all within app?

We might have a duplicate issue for this, i'll have to look. It was something we said we'd consider for 2.0 IIRC

**Comment by alimeerutech** [comment]
correct, that is the case. We have multiple apps sharing the same context
variables. thanks

**Comment by Cloudrage** [comment]
+1
Or that way : https://github.com/aws/aws-cdk/issues/4543

**Comment by hannonq** [comment]
+1

**Comment by hkhrais** [comment]
+1

**Comment by bert-bae** [comment]
+1

**Comment by github4es** [comment]
+1

**Comment by ns-ajith** [comment]
+1

**Comment by sasinda** [comment]
+1

**Comment by brianarpie-forme** [comment]
+1

**Comment by cbou** [comment]
+1

**Comment by cpalmer-ostmodern** [comment]
Would it be easier to use a mono repo with a shared cdk folder ?

**Comment by MathiasHaudgaard** [comment]
any updates?

**Comment by sigpop** [comment]
This would be useful.

**Comment by ribase** [comment]
Anything going on here, this feature still would be nice to have...
